invenio_vocabularies/assets/semantic-ui/js/invenio_vocabularies/src/contrib/forms/Funding/FundingModal.js

```diff
@@ -157,8 +157,8 @@ function FundingModal({
         >
           <Modal.Header as="h2" className="pt-10 pb-10">
             {mode === "standard"
-              ? i18next.t("Add standard award")
-              : i18next.t("Add custom award")}
+              ? i18next.t("Add standard award/grant")
+              : i18next.t("Add custom award/grant")}
```

Comment:
Maybe "Add custom funding"? Since only the Funder is required, you don't actually have to provide information about a award or grant.

Reply:
Good point, thanks! Done :+1: